specs/merge/beacon-chain.md

```diff
@@ -59,148 +57,138 @@ We define the following Python custom types for type hinting and readability:
 | Name | Value |
 | - | - |
 | `MAX_BYTES_PER_OPAQUE_TRANSACTION` | `uint64(2**20)` (= 1,048,576) |
-| `MAX_EXECUTION_TRANSACTIONS` | `uint64(2**14)` (= 16,384) |
+| `MAX_OPAQUE_TRANSACTIONS` | `uint64(2**14)` (= 16,384) |
```

Comment:
See my separate comment. These are *transactions*. Not only opaque ones. It's just that opaque is the only type currently.

Maybe just call them `Transactions` -- `MAX_TRANSACTIONS`. We reserved "operation" for validator/system level block messages to not collide with "transaction" as user layer

Reply:
I've added a new `Transaction` type placeholder to be defined using a union type. Renamed to `MAX_TRANSACTIONS_PER_PAYLOAD`.